octopoes/octopoes/events/manager.py

```diff
@@ -26,6 +26,7 @@ class ScanProfileMutation(BaseModel):
     operation: OperationType
     primary_key: str
     value: AbstractOOI | None = None
+    client_id: str
```

Comment:
I named this `client_id`  since organisations are referred to as clients in octopoes. So not sure if this should be organisation or client